code/index.html

```diff
@@ -2,21 +2,107 @@
       <!-- The other news  -->
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/200" alt="small news">
+        <h3 class="small-news-title">X Reasons Why</h2>
+        <p class="small-news-text">
+          Three-martini lunch quick win, nor we need to harvest synergy effects but problem territories. Killing it can you send me an invite?
+        </p>
+      </article>
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/201" alt="small news">
+        <h3 class="small-news-title">This Is What</h2>
+        <p class="small-news-text">
+          Show pony action item Bob called an all-hands this afternoon, for streamline, for pre launch. Code we need to get all stakeholders up to speed.
+        </p>
+      </article>
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/202" alt="small news">
+        <h3 class="small-news-title">You Can Now</h2>
+        <p class="small-news-text">
+          Can you champion this spinning our wheels globalize. Can we align on lunch orders back to the drawing-board, nor draw a line in the sand.
+        </p>
+      </article>
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/203" alt="small news">
+        <h3 class="small-news-title">This Is How</h2>
+        <p class="small-news-text">
+          What you've given us is texty, we want sexy can you please send me the design specs again? So can you make it more infographic-y, make it wiggle.
+        </p>
+      </article>
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/204" alt="small news">
+        <h3 class="small-news-title">X Things You</h2>
+        <p class="small-news-text">
+          Draft policy ppml proposal criticality for knowledge is power. Data-point product market fit so copy and paste from stack overflow.
+        </p>
+      </article>
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/205" alt="small news">
+        <h3 class="small-news-title">You Can Now</h2>
+        <p class="small-news-text">
+          What is lorem ipsum? Why is the text in spanish? Can you use a high definition screenshot start on it today and we will talk about what i want next time.
+        </p>
+      </article>
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/206" alt="small news">
+        <h3 class="small-news-title">The Last</h2>
+        <p class="small-news-text">
+          Anyway, you are the designer, you know what to do try making it a bit less blah can you put "find us on facebook" by the facebook logo?
+        </p>
+      </article>
     </section>
+
+    <footer>
+      <div class="footer">
```

Comment:
The footer itself can be style in css without having to put another div. As you did in CSS, your selector is footer. So no need of `<div class="footer">` .